nstars.c

```diff
@@ -157,7 +157,7 @@ int main(int argc, char *argv[])
 					beep();
 				}
 				break;
-			case 10:					/* Enter */
+			case KEY_ENTER:
```

Comment:
re: slash escapes -

static chars are ints, right? As long as they're equivalent, I think that's fine. You think that way will be more readable in the code?

re: platform

I run Debian Jessie and xterm. I'm not familiar with what "st" is..